cfme/tests/infrastructure/test_snapshot.py

```diff
@@ -129,7 +126,7 @@ def test_snapshot_crud(small_test_vm, provider):
 
 
 @pytest.mark.rhv3
-@pytest.mark.uncollectif(lambda provider: not provider.one_of(RHEVMProvider))
+@pytest.mark.provider([RHEVMProvider], override=True)
```

Comment:
same question about the uncollect.

Reply:
uncollecting against _not_ rhevm, now marking for rhevm